evm/utils.js

```diff
@@ -554,6 +560,54 @@ const deployContract = async (
     }
 };
 
+/**
+ * Validate if the input string matches the time format YYYY-MM-DDTHH:mm:ss
+ *
+ * @param {string} timeString - The input time string.
+ * @return {boolean} - Returns true if the format matches, false otherwise.
+ */
+function isValidTimeFormat(timeString) {
+    const regex = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}$/;
```

Comment:
This regex can also pass invalid date time like for ex- `2023-08-10T80:70:70` Here hours is 80, minutes 70, seconds 70 which will pass but is wrong.
We would need to modify this.

Reply:
Good catch, I updated the regex to validate months, days, hours, minutes, and seconds. Please take a look